packages/ag-charts-community/src/axis.ts

```diff
@@ -151,7 +151,7 @@ export abstract class Axis<S extends Scale<D, number, TickInterval<S>> = Scale<a
     linkedTo?: Axis<any, any>;
     includeInvisibleDomains: boolean = false;
 
-    readonly axisGroup = new Group({ name: `${this.id}-axis`, zIndex: Layers.AXIS_ZINDEX });
+    readonly axisGroup = new Group({ name: `${this.id}-axis`, zIndex: 10000 });
```

Comment:
Nit: Please keep the convention of using `Layers` for all significant `zIndex` values.